index.js

```diff
@@ -814,6 +853,45 @@ Client.prototype.getChildren = function (path, watcher, callback) {
     );
 };
 
+/**
+ * BFS list of the system under path. Note that this is not an atomic snapshot of
+ * the tree, but the state as it exists across multiple RPCs from clients to the
+ * ensemble.
+ *
+ * @method getAllChildren
+ * @param path {String} The node path.
+ * @param callback {Function} The callback function.
+ */
+Client.prototype.getAllChildren = function(path, callback) {
```

Comment:
Why do we need to expose this method? Again, the the `listSubTreeBFS` is a better name even for private method.